frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -1080,6 +1080,9 @@ private void prepareReleaseDuplicate(RenderRequest request, RenderResponse respo
             Release release = PortletUtils.cloneRelease(emailFromRequest, client.getAccessibleReleaseById(releaseId, user));
             Map<String, String> sortedAdditionalData = getSortedMap(release.getAdditionalData(), true);
             release.setAdditionalData(sortedAdditionalData);
+            release.unsetModifiedBy();
+            release.unsetModifiedOn();
+            release.unsetAdditionalData();
```

Comment:
This isn't the right place, Please unset all the attributes in `cloneProject` method of  `PortletUtils` class.
screenshot for reference:
![image](https://user-images.githubusercontent.com/50870174/206685536-ac58382f-15b2-4946-b34e-46d55b9b6598.png)

Reply:
done.